processing.py

```diff
@@ -0,0 +1,109 @@
+# processing Carvana image dataset
+# All input data should be kept in a folder called 'input'
+
+import skimage
+from skimage import data
+from scipy import ndimage
+import numpy as np
+import pandas as pd
+from pandas import DataFrame
+import matplotlib.pylab as plt
+import os
+from glob import glob
+
+INPUT_PATH = '../input' #Get the paths to the input data
+DATA_PATH = INPUT_PATH
+
+TRAIN_DATA = os.path.join(DATA_PATH, "train")
+TRAIN_MASKS_DATA = os.path.join(DATA_PATH, "train_masks")
+#TEST_DATA = os.path.join(DATA_PATH, "test") #comment out until actual testing
+TRAIN_MASKS_CSV_PATH = os.path.join(DATA_PATH, "train_masks.csv")
+METADATA_CSV_PATH = os.path.join(DATA_PATH, "metadata.csv")
+
+TRAIN_MASKS_CSV = pd.read_csv(TRAIN_MASKS_CSV_PATH) #Read DataFrames
+METADATA_CSV = pd.read_csv(METADATA_CSV_PATH)
+
+train_files = glob(os.path.join(TRAIN_DATA, "*.jpg")) #Get paths for each image
+train_ids = [s[len(TRAIN_DATA)+1:-4] for s in train_files] #Get the id's by removing ../input/train/ and .jpg
+
+"""
+# Get the test image files and their id's into lists
```

Comment:
Why is there a comment in a comment block?

Reply:
I just commented out that whole block for the same reason above.